build.sbt

```diff
@@ -269,6 +269,8 @@ lazy val assemblySettings = assembly / assemblyMergeStrategy := {
 
 Test / parallelExecution := true
 
+core / Compile / logLevel := Level.Error
```

Comment:
I don't think we should do this. Because then we will miss out on deprecation logs. Instead we have a ticket in current sprint to address all warn messages.

Reply:
this currently saves 1 minute of building time with it present, maybe i can just add it during the ci cd somehow

Reply:
okay i figured it out, i can append this to the file before the build